specification/advisor/resource-manager/Microsoft.Advisor/stable/2020-01-01/advisor.json

```diff
@@ -858,13 +858,18 @@
       "description": "Advisor Digest configuration entity",
       "type": "object",
       "properties": {
+        "name": {
+          "description": "Name of digest configuration. Value is case-insensitive and must be unique within a subscription.",
+          "type": "string"
+        },
         "actionGroupResourceId": {
           "description": "Action group resource id used by digest.",
           "type": "string"
         },
         "frequency": {
-          "description": "Frequency that digest will be triggered. Value must conform to ISO 8601 standard and must be greater than equal to 7 day and less than or equal to 30 days.",
-          "type": "string"
+          "format": "int32",
```

Comment:
I think the empty line return failed prettierCheck. Please ensure all Checks are passing.